CMakeLists.txt

```diff
@@ -243,6 +244,7 @@ if(OXENMQ_INSTALL)
             lokimq/bt_serialize.h
             lokimq/bt_value.h
             lokimq/connections.h
+            lokimq/formattable.h
```

Comment:
Shouldn't be here; these are only the backwards-compat shims for the pre-renamed lokimq headers.

Reply:
(I'll probably drop them completely one of these days)